Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -346,6 +410,36 @@ script:
       - 'Enabled'
       - 'Suspended'
       required: true
+
+  - name: aws-s3-bucket-versioning-enable-quick-action
+    description: Enables versioning on the S3 bucket to retain multiple versions of objects by running the aws s3api put-bucket-versioning command. Successful execution requires the s3:PutBucketVersioning permission.
```

```suggestion
    description: Enables versioning on the S3 bucket to retain multiple versions of objects by running the aws s3api put-bucket-versioning command.
```
The same here, please replace aws s3api put-bucket-versioning.